docs-content/release-notes.html.md.erb

```diff
@@ -3,10 +3,16 @@ title: Release Notes for VMware Tanzu Observability by Wavefront Nozzle
 owner: Partners
 ---
 
-<strong>Page last updated: November 18, 2021</strong>
+<strong>Page last updated: December 10, 2021</strong>
```

Comment:
Last updated date is the date on which the document is edited. Let's put the date on which the doc will be pushed out.